packages/webapp/src/pages/Connection/Authorization.tsx

```diff
@@ -4,14 +4,14 @@ import { Loading } from '@geist-ui/core';
 
```

Comment:
When I got in this tab with an error I get `Cannot read properties of undefined (reading 'type')` 
edit: might be due to a 500 actually (I just broke the credentials to create an error :D)

Reply:
Added fallback for type if it is missing which in `unauthenticated` provider it is